docs/components/Markdown/index.tsx

```diff
@@ -0,0 +1,27 @@
+import hljs from 'highlight.js';
+import 'highlight.js/styles/a11y-dark.css';
+import { marked } from 'marked';
+import { observer } from 'mobx-react';
+import * as React from 'react';
+
+interface Props {
+  markdown: string;
```

Comment:
Since we're dealing with potential for XSS, I'd prefer we type this to something more than just a `string`. That way it's clear when we might be wanting to feed unsafe content and it makes it more explicit when it's read as "safe".

```ts
interface Markdown {
  content: string;
}

const markdown = (): Markdown => ___;
```

Reply:
Ah, good point. Kind of like how we use decoders to verify types at the edge of the app, this way we verify "safety" at the edge of the app.